<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PathWindow</class>
 <widget class="QWidget" name="PathWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>武汉大学导航系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#PathWindow {
	background-color: #fff8eb;
}
QLabel {
	font-size: 20px;
	font-family: consolas, 黑体;
}
QPushButton{
	padding: 2px 12px 2px 12px;
	font-family: 微软雅黑;
    font-size: 18px;
    color: white;
    background-color: rgb(222,184,135);
    border: 2px solid rgb(205,133,63);
    border-radius: 5px;
}
QPushButton:hover{
	background-color: rgb(210,174,137);
}
#titleLabel{
	background-color: rgb(161, 117, 15);
	color: white;
	font-size: 36px;
	font-family: 华文行楷;
	padding: 10px auto 5px 5px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,15">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string> 武汉大学校园导航系统</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3,1,3,1,2,1,1,1">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="srcLabel">
       <property name="text">
        <string>起点：湖滨十一舍</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="dstLabel">
       <property name="text">
        <string>终点：文理学部第五教学楼</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="algLabel">
       <property name="text">
        <string>算法：Dijkstra</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="returnButton">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWebEngineView" name="web" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>PySide2.QtWebEngineWidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
